packaging/windows/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian
+RUN apt-get update && \
+    apt-get install -y nsis && \
+    apt-get install -y wget && \
+    apt-get install -y unzip
+RUN mkdir app    
+WORKDIR /home
+RUN wget https://nssm.cc/release/nssm-2.24.zip
```

Comment:
There's several options that people use by default when installing things via `curl` that help ensure it's a bit more secure. I assume there's something similar for `wget` (enforce TLS 1.2, strict cert checking, etc). You can find a curl-specific example here: https://rustup.rs/

Reply:
Maybe SHA256 checking as well?